forest/shared/Cargo.toml

```diff
@@ -8,66 +8,35 @@ edition = "2021"
 [dependencies]
 anyhow.workspace = true
 atty.workspace = true
-base64.workspace = true
 byte-unit = "4.0"
 chrono.workspace = true
-cid.workspace = true
 console-subscriber.workspace = true
 directories.workspace = true
-fil_actor_miner_v8.workspace = true
-forest_actor_interface.workspace = true
-forest_beacon.workspace = true
-forest_blocks.workspace = true
-forest_chain.workspace = true
 forest_chain_sync.workspace = true
-forest_crypto.workspace = true
 forest_db.workspace = true
-forest_deleg_cns = { workspace = true, optional = true }
-forest_encoding.workspace = true
-forest_fil_types.workspace = true
-forest_interpreter.workspace = true
-forest_json.workspace = true
-forest_key_management.workspace = true
 forest_libp2p.workspace = true
-forest_message = { workspace = true, features = ["blst"] }
-forest_message_pool.workspace = true
 forest_networks.workspace = true
-forest_rpc-api.workspace = true
 forest_rpc-client.workspace = true
-forest_state_manager.workspace = true
 forest_utils.workspace = true
 futures.workspace = true
-fvm.workspace = true
-fvm_ipld_car.workspace = true
-fvm_ipld_encoding.workspace = true
 fvm_shared = { workspace = true, default-features = false }
 gethostname.workspace = true
 git-version.workspace = true
 hex.workspace = true
-jsonrpc-v2.workspace = true
-lazy_static.workspace = true
-libp2p = { workspace = true, default-features = false, features = ["identify"] }
 log = { workspace = true, features = ["serde"] }
-multibase.workspace = true
-num-rational.workspace = true
-num-traits.workspace = true
 once_cell.workspace = true
-prometheus = { workspace = true, features = ["process"] }
 regex.workspace = true
-rust-s3 = { version = "0.32", default-features = false, features = ["sync-rustls-tls", "tags"] }
+s3 = { package = "rust-s3", version = "0.32", default-features = false, features = ["sync-rustls-tls", "tags"] }
```

Comment:
This is interesting. I'm surprised it worked before.

Reply:
The crate [specifies `s3` as lib name](https://docs.rs/crate/rust-s3/latest/source/Cargo.toml.orig), this change is to avoid false positive from the script cuz it's using a very simple regex rule